google/resource_container_cluster.go

```diff
@@ -1490,6 +1525,23 @@ func expandClusterAddonsConfig(configured interface{}) *containerBeta.AddonsConf
 		}
 	}
 
+	if v, ok := config["istio_config"]; ok && len(v.([]interface{})) > 0 {
+		addon := v.([]interface{})[0].(map[string]interface{})
+		ac.IstioConfig = &containerBeta.IstioConfig{
+			Disabled:        addon["disabled"].(bool),
+			Auth:            addon["auth"].(string),
+			ForceSendFields: []string{"Disabled", "Auth"},
+		}
+	}
+
+	if v, ok := config["cloudrun_config"]; ok && len(v.([]interface{})) > 0 {
+		addon := v.([]interface{})[0].(map[string]interface{})
+		ac.IstioConfig = &containerBeta.IstioConfig{
```

Comment:
Not familiar with what cloudrun refers to, but is this supposed to be modifying the IstioConfig here?

Reply:
you're right. CloudRun is a feature on top of Istio (CloudRun depends on Istio in container/v1beta1 API). Fixed, thank you